xtask/src/main.rs

```diff
@@ -98,7 +98,7 @@ pub fn main() -> cross::Result<()> {
             target_info::target_info(args, &engine, &mut msg_info)?;
         }
         Commands::BuildDockerImage(args) => {
-            let mut msg_info = get_msg_info!(args, args.verbose != 0)?;
```

Comment:
We now want to add the `VERBOSE` flag with `-vv` builds, I believe, so we should re-enable this to accept the levels, unless there's a reason I'm forgetting.

Reply:
I don't think there is a reason, just an artifact when -vv was enabled